GuardedView.fs

```diff
@@ -319,6 +349,16 @@ let pruneGuardedSet (gset : Multiset<Guarded<_, _>>)
                    | _       -> true)
     |> Multiset.ofFlatSeq
 
+/// Gets a Set of TypedVar's from an Expr
+let varsFromExpr : Expr<Sym<Var>> -> Set<TypedVar> =
+    function
+    | Bool e -> mapOverSymVars Mapper.mapBoolCtx findSymVars e
+    | Int e -> mapOverSymVars Mapper.mapIntCtx findSymVars e
+
+/// Gets set of TypedVar's from a SVFunc
+let gfuncVars : SVFunc -> Set<TypedVar> =
+    fun gf -> Set.fold (+) Set.empty (Set.ofList (List.map varsFromExpr gf.Params))
```

Comment:
Set.unionMany is the correct one I think.  In my changeset-to-come, I got a bit carried away fixing these…